scripts/release.sh

```diff
@@ -2,14 +2,33 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+function is_gnu_sed() {
+  sed --version >/dev/null 2>&1
+}
+
+function find_and_replace() {
+    EXPRESSION="$1"
+    FILE="$2"
+    if is_gnu_sed; then
+        sed -i -r "$EXPRESSION" "$FILE"
+    else
+        sed -i '' -E "$EXPRESSION" "$FILE"
+    fi
+}
+
 cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 
 CURRENT=$(grep -E -o '[0-9]' ./internal/gorson/version/version.go)
```

Comment:
this is pretty interesting... this doesn't work once we get to version 10

```
1
0
```

we should update this to:

```
CURRENT=$(grep -E -o '[0-9].' ./internal/gorson/version/version.go)
```

Reply:
Good catch! I changed it to:

```bash
CURRENT=$(grep -E -o '[0-9]+' ./internal/gorson/version/version.go)
```

as that captures any natural number version without capturing quotes.